src/modules/grubcfg/main.py

```diff
@@ -23,6 +23,8 @@
 import os
 import re
 
+def setExpression(pattern, file):
+        libcalamares.utils.target_env_call(["sed", "-e", pattern, "-i", file])
```

Comment:
I think you might have an alignment issue caused by your IDE? We use 4 spaces and no tabs in Calamares.